atlasapi/measurements.py

```diff
@@ -0,0 +1,422 @@
+from __future__ import division, absolute_import, print_function, unicode_literals
+
+from datetime import datetime
+from statistics import mean, StatisticsError
+from typing import Optional, Tuple, List, Iterable, NewType
+import logging
+import humanfriendly as hf
+from dateutil.parser import parse
+
+from atlasapi.atlas_types import OptionalFloat
+from atlasapi.lib import _GetAll, AtlasPeriods, AtlasGranularities
+
+logger: logging.Logger = logging.getLogger('Atlas.measurements')
+
+
+class StatisticalValues:
+    def __init__(self, data_list: list):
+        try:
+            self.samples: int = len(clean_list(data_list))
+            self.mean: float = float(mean(clean_list(data_list)))
+            self.min: float = float(min(clean_list(data_list)))
+            self.max: float = float(max(clean_list(data_list)))
+        except StatisticsError:
+            logger.warning('Could not compute statistical values.')
+            self.samples: int = 0
+            self.mean: int = 0
```

Comment:
mean as a float for consistency with the declaration above ^^

Reply:
Fixed! New commit coming.